src/components/LookupMobileProofing.js

```diff
@@ -1,42 +1,56 @@
 import React, { Component } from "react";
 import PropTypes from "prop-types";
 import NotificationModal from "../login/components/Modals/NotificationModal";
-
-class LookupMobileProofing extends Component {
+import { HashLink } from 'react-router-hash-link';
+class LookupMobileProofing extends Component {  
   render() {
+    /* Description is the text inside the vetting button */
     let description = "";
+    /* without nin, description text will help the user to add id number */
     if(this.props.withoutNin){
       description = (
         <div className="description">
           {this.props.translate("verify-identity.vetting_explanation_add_nin")}
         </div> 
-      )
+      ) /* without phone number, description text will help the user to add phone number and the text "setting" is linked to the setting page phone number section */ 
     } else if(this.props.withoutPhoneNumber){
       description = (
-        <div className="description">
+        <div className="link">
           {this.props.translate("verify-identity.vetting_explanation_add_phone_number")}
+          <HashLink
+            key="1"
+            to={"/profile/settings/#phone"}
+          >
+            {this.props.translate("verify-identity.vetting_link_settings")}
+          </HashLink>
         </div> 
-      )
+      ) /* without verified phone number, description text will help the user to confirm phone number and the text "setting" is linked to the setting page phone number section */
     } else if(this.props.notVerifiedNumber) {
```

Comment:
we have previously used "unverified" to describe data that has not gone through a verification or confirmation process

Reply:
I have changed the name to align with previous used wording 
Now "unverifiedNumber"